### ISSUE TITLE
basic extension removes <u>

### ISSUE THREAD

**Issue by pguetschow** [tgalopin/html-sanitizer-bundle#23]
According to the library documentation, <u> should not be removed when using the basic extension 

> basic allows the insertion of basic HTML elements: a, b, br, blockquote, div, del, em, figcaption, figure, h1, h2, h3, h4, h5, h6, i, p, q, small, span, strong, sub, sup, u

but in my case, it is removed. 

My Service Config:
```
html_sanitizer:
    default_sanitizer: 'default'
    sanitizers:
        default:
            extensions:
                - 'basic'
                - 'list'
                - 'table'
```

Extending the InputType
```
    App\Form\Extension\TextInputTypeExtension:
        arguments:
            $sanitizer: '@html_sanitizer.default'
```
Inside the Extension
```
public function buildForm(FormBuilderInterface $builder, array $options): void
    {
        $builder->addEventListener(FormEvents::PRE_SUBMIT, function (FormEvent $event) use ($builder) {
            if (null != $event->getData()) {
                $data = $this->sanitizer->sanitize($event->getData());
                $event->setData(trim($data));
            }
        });
    }
```
If I dump `$event->getData()` I get the string containing the `<u>`, eg. `<p><strong><u>Registrierung</u> und Anmeldung</strong></p>` but after the sanitizing, `$data` just resolves to `<p><strong>Registrierung und Anmeldung</strong></p>`

Any idea why?

**Comment by tgalopin** [comment]
Indeed, this was not released it seems; let me release it :) .

**Comment by tgalopin** [comment]
This should now be fixed (released as 1.5.0)

**Comment by pguetschow** [comment]
Thanks @tgalopin, looking forward to the version
